auth.proto

```diff
@@ -6,21 +6,26 @@ syntax = "proto3";
 package magistrala;
 option go_package = "./magistrala";
 
-// AuthzService is a service that provides authentication and authorization
+// ThingsService is a service that provides authentication and authorization
 // functionalities for the things service.
-service AuthzService {
+service ThingsService {
```

Comment:
Let's keep the `Authz` name. While this is Things in the implemetation - this is not implementation-oriented (i.e. caller should not care).